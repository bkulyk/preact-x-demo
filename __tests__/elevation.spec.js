import React from 'react';
import Elevation from '@bitchin/elevation';
import { shallow } from './enzyme';
import { it, expect } from './jest';

it('renders correctly, with default z prop', () => {
  expect(shallow((
    <Elevation
      title="some-title"
      url="http://example.com/awesome-pic.png"
      className="mine"
    >
      content
    </Elevation>
  ))).toMatchSnapshot();
});

it('renders correctly, with z prop', () => {
  expect(shallow((
    <Elevation
      title="some-title"
      url="http://example.com/awesome-pic.png"
      className="mine"
      children="content"
      z="12"
    />
  ))).toMatchSnapshot();
});
