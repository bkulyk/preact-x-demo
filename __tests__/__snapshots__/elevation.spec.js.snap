// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly, with default z prop 1`] = `
<div
  className="mdc-elevation-transition mdc-elevation--z1 mine"
  title="some-title"
  url="http://example.com/awesome-pic.png"
>
  content
</div>
`;

exports[`renders correctly, with z prop 1`] = `
<div
  className="mdc-elevation-transition mdc-elevation--z12 mine"
  title="some-title"
  url="http://example.com/awesome-pic.png"
>
  content
</div>
`;
